test/integration/043_custom_aliases_test/models/schema.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+
+models:
```

Comment:
nice! Can you add another model + test which _does not_ provide an alias? The resulting `this.name` should just be the name of the model.